[source-redshift]:  driver updates and sql updates to include late binding views

nisargpatel-m opened the pull request:
- Bump Redshift JDBC driver version to 2.1.0.32.
- Add AWS SDK core dependency for future compatibility.
- Refactor SQL query in RedshiftSource to include views.

msaffitz commented on airbyte-integrations/connectors/source-redshift/src/main/java/io/airbyte/integrations/source/redshift/RedshiftSource.java:
```diff
@@ -115,9 +115,15 @@ public Set<JdbcPrivilegeDto> getPrivilegesTableForCurrentUser(final JdbcDatabase
-                  + "WHERE  has_table_privilege(schemaname||'.'||tablename, 'select') = true AND schemaname = ?;");
+              "select schemaname, tablename "
+                + "from (SELECT "
+                + "  v.schemaname as schemaname, "
+                + "  v.resourcename as tablename "
+                + "FROM "
+                + "  (SELECT schemaname, viewname as resourcename FROM pg_views "
+                + "  UNION ALL SELECT schemaname, tablename as resourcename from pg_tables) v "
+                + "WHERE "
+                + "    has_table_privilege(current_user, v.schemaname || '.' || quote_ident(v.resourcename), 'select') = true and v.schemaname = ?) as permission;");
```
do you need all of the `as ....` in there given you're not really renaming things?  seems like maybe only the `viewname as tablename` may strictly be needed and might make readability a little better.  It also seems like the outer select could be removed (not sure-- with the formatting here it's hard to see-- it looks to me like this is doing `select schemaname, tablename from (...) as permission;` which I don't think is valid?

nisargpatel-m replied:
Yeah; fair point. Im going to remove the outer query as we are not doing any further joins/selects on it.

msaffitz reviewed (approved):
great work!  one question on the query; looks good to merge if that's just confusion on my part :)

nisargpatel-m committed:
Update Redshift JDBC dependency and enhance table selection query.

- Bump Redshift JDBC driver version to 2.1.0.32.
- Add AWS SDK core dependency for future compatibility.
- Refactor SQL query in RedshiftSource to include views.

mxf9802 reviewed (approved):
:shipit:
